cumulusci/cli/cci.py

```diff
@@ -895,9 +895,12 @@ def org_info(config, org_name, print_json):
         ]
         keys = [key for key in org_config.config.keys() if key in UI_KEYS]
         keys.sort()
+        pairs = [[key, str(org_config.config[key])] for key in keys]
+        pairs.append(["release", str(org_config.latest_api_version)])
```

Comment:
I would call this `api_version`. As it stands API versions are incremented at the same time that releases are made, but I would expect "release" to be either the release number (e.g. 222) or customer-facing release name like Winter '20.

Reply:
You are just referring to this line, right? The name OrgConfig.latest_api_version is fine?